webapp/channels/src/components/get_public_link_modal/get_public_link_modal.test.tsx

```diff
@@ -3,10 +3,13 @@
 
 import {shallow} from 'enzyme';
 import React from 'react';
+import {act} from 'react-dom/test-utils';
 
 import GetLinkModal from 'components/get_link_modal';
 import GetPublicLinkModal from 'components/get_public_link_modal/get_public_link_modal';
 
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
```

Comment:
Any strong reason to use `mountWithIntl` instead of `shallowWithIntl`? It used to have shallow instead of mount.

Reply:
`useEffect` is not supported by enzyme `shallow` render that is the reason I shifted the test case to `mount`. [ref](https://stackoverflow.com/questions/59164027/testing-a-component-that-uses-useeffect-using-enzyme-shallow-and-not-mount#answer-59174911)